imap_processing/lo/l0/lol0.py

```diff
@@ -0,0 +1,48 @@
+"""General Lo L0 data class used for parsing data and setting attributes."""
+from dataclasses import dataclass, fields
+
+from imap_processing.ccsds.ccsds_data import CcsdsData
+
+
+@dataclass
+class LoL0:
+    """Data structure for common values across histogram and direct events data.
+
+    Attributes
+    ----------
+    ground_sw_version : str
+        Ground software version
+    packet_file_name : str
+        File name of the source packet
+    ccsds_header : CcsdsData
+        CCSDS header data
+    """
+
+    ground_sw_version: str
```

Comment:
Rather than passing it in here, it probably works better to put it in `__init__.py` and import that version when needed